lib/parallel_tests/cli.rb

```diff
@@ -2,6 +2,11 @@
 require 'tempfile'
 require 'parallel_tests'
 require 'shellwords'
+require_relative 'process_handler'
+
+# When this script receives the SIGINT signal from ctrl+c, send the SIGINT
+# signal to all child processes
+Signal.trap(:INT) { ParallelTests::ProcessHandler.kill_child_processes }
```

Comment:
Yeah I'll move that out. The reason I added it was because ctrl+c was killing the direct child processes but the grandchild processes kept attempting to write output resulting in Broken pipe @ io_write - <STDOUT> (Errno::EPIPE)